lib/blocs/authentication/viewmodels/authentication_bloc.dart

```diff
@@ -40,6 +41,12 @@ class AuthenticationBloc extends Bloc<AuthenticationEvent, AuthenticationState>
       // Recovered account --> re-start auth status
       add(const InitAuthStatus());
     }
+    if (event is OnSwitchAccount) {
+      SwitchAccountUseCase().run(event.account);
+      settingsStorage.privateKeyBackedUp = true;
```

Comment:
this should be in a use case. 
And is there no method to save this for us in the `settingsStorage` ?

Reply:
> this should be in a use case.

You mean the settingsStorage.privateKeyBackedUp?

> And is there no method to save this for us in the settingsStorage ?

To update the flag privateKeyBackedUp ?

Reply:
yes and yes

Reply:
done